packages/integration-tests/cmr.js

```diff
@@ -105,8 +106,8 @@ async function waitForConceptExistsOutcome(cmrLink, expectation, retries = 3, in
         timeout: interval * retries
```

Comment:
Retries is set to 3, but Chris Durbin mentioned that he sees rare occasions where it takes up to 40 seconds to ingest to UAT.  Should we change the interval & retries values to something to cover that time?

Reply:
Bumping the timeout to 60 seconds seems reasonable.

Reply:
Agree